docs/src/extend/custom-rules.md

```diff
@@ -106,20 +105,36 @@ module.exports = {
 
 ## The Context Object
 
-The `context` object contains additional functionality that is helpful for rules to do their jobs. As the name implies, the `context` object contains information that is relevant to the context of the rule. The `context` object has the following properties:
+The `context` object is the argument of the callback function in the `create` property of a custom rule. For example:
 
-* `parserOptions` - the parser options configured for this run (more details [here](../use/configure/language-options#specifying-parser-options)).
-* `id` - the rule ID.
-* `options` - an array of the [configured options](../use/configure/rules#configuring-rules) for this rule. This array does not include the rule severity. For more information, see [here](#contextoptions).
-* `settings` - the [shared settings](../use/configure/configuration-files#adding-shared-settings) from configuration.
-* `parserPath` - the name of the `parser` from configuration.
-* `parserServices` - an object containing parser-provided services for rules. The default parser does not provide any services. However, if a rule is intended to be used with a custom parser, it could use `parserServices` to access anything provided by that parser. (For example, a TypeScript parser could provide the ability to get the computed type of a given node.)
+```js
+// customRule.js
+
+module.exports = {
+    meta: { ... },
+    // `context` object is the argument
+    create: function(context) {
+       // ...
+    }
+};
+```
+
+The `context` object contains additional functionality that is helpful for rules to do their jobs. As the name implies, the `context` object contains information that is relevant to the context of the rule.
+
+The `context` object has the following properties:
+
+* `parserOptions`: The parser options configured for this run (more details [here](../use/configure/language-options#specifying-parser-options)).
```

Comment:
parserOptions is deprecated now, so we should move this into a separate area.

Reply:
moving to ``### Deprecated `context` Methods`` section